pykeepass/pykeepass.py

```diff
@@ -209,6 +212,11 @@ def transformed_key(self):
         and passed to `open` for faster database opening"""
         return self.kdbx.body.transformed_key
 
+    @property
+    def master_seed(self):
+       """bytes: master seed of database"""
```

Comment:
Is there more that can be said about this? e.g. is there an expected size?

Reply:
It's no longer the master_seed but the salt and if i remember correctly this is not fixed. Any idea?